requirments/REQUIREMENTS.md

```diff
@@ -0,0 +1,35 @@
+## Roles
+#### Unauthorized user
+Unathorized users can view books and comments to them, but cannot purchase books or subscriptions. Also they can sign in.
+#### Authorized user
+Authorized users can view books, purchase books and subscriptions, view and leave comments. Also they can sign out.
+#### Moderator
+Moderators can view, edit, leave and remove comments to books.
+#### Copyright holder
+Copyright holders can agree with a service manager to add or remove a book to/from the service.
+#### Service manager
+Service managers have contacts of copyright holders and can agree with them to add/remove books.
+
+## Use case
+![use cases](https://github.com/kkarnauk/se-project-fall-2021/blob/task_6/img/useCase.jpg)
+## User stories
+
+#### User story №1
+As user, I want to add books to cart, so I can buy them
+
+#### User story №2
+As moderator, I want to delete book reviews in order to remove offencive texts from service
+
+#### User story №3
+As service manager, I want to sign agreement with copyright holders, so I can add their books to service
+
+## Job stories
```

Comment:
I am not sure you understand correctly the way Job Stories work. Those stories about some actors in your system that are in some situation and wish to change that situation to achieve something. In your case, your job stories about some ephemeral entity that wants to organize the world…